ts/features/bonus/bpd/saga/orchestration/optInPaymentMethods/optInShouldShowChoiceHandler.ts

```diff
@@ -38,8 +38,13 @@ export function* optInShouldShowChoiceHandler(): Generator<
   void,
   any
 > {
-  // Load the information about the participation of the user to the bpd program
-  yield* put(bpdLoadActivationStatus.request());
+  const iactivationInitialStatus: RemoteValue<ActivationStatus, Error> =
+    yield* select(activationStatusSelector);
+
+  if (!isLoading(iactivationInitialStatus)) {
```

Comment:
```suggestion
  // Check is needed to avoid to spawn multiple request if the status
  // is already loading
  if (!isLoading(bpdActivationInitialStatus)) {
```